recipes/glad/2.x/test_package/test_package.c

```diff
@@ -0,0 +1,24 @@
+#include <glad/gl.h>
+#include <GLFW/glfw3.h>
+
+#include <stdio.h>
+
+int main() {
+    GLFWwindow* window;
+
+    #ifdef __APPLE__
+    if (!glfwInit()) {
+        return -1;
+    }
+    #endif
+
+    window = glfwCreateWindow(1, 1, "Hello World", NULL, NULL);
```

Comment:
Please do not initialise UI, or interact with the filesystem or network in any test package, thanks!